awx_collection/plugins/modules/tower_credential_input_source.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright: (c) 2017, Wayne Witzel III <wayne@riotousliving.com>
```

Comment:
Nitpick: I would suggest putting "Tom Page (@Tompage1994)" here as well, to reflect what it says in the `DOCUMENTATION` section below.